### ISSUE TITLE
Add Dharman, Ryan M; update Tim Post

### ISSUE THREAD

**Issue by blackgreen100** [Oaphi/mods-tex#1]
Might need an update: Dharman and Ryan M's rows are missing. Tim Post also resigned [around July 11th](https://chat.stackoverflow.com/transcript/41570?m=54898647#54898647). Thank you!

**Comment by Oaphi** [comment]
Yeah, definitely - given it's not automatic, I keep forgetting to update it. Will do
